src/bitcoin/OrdinalsClient.ts

```diff
@@ -45,6 +35,96 @@ export class OrdinalsClient {
   async estimateFee(blocks: number = 1): Promise<number> {
```

Comment:
@cursoragent this doesn't seem to be the real implementation. You can find this in the ordinalsplus package